app/BasicViewController.m

```diff
@@ -22,6 +22,7 @@
 #import <GoogleMaps/GoogleMaps.h>
 #import <GoogleMapsUtils/GoogleMapsUtils.h>
 #import "POIItem.h"
+#import "GoogleMapsUtils-Swift.h"
```

Comment:
Remove?

Reply:
I think these auto-generated for some reason. I removed them, but I'm not sure if they'll reappear if the framework is built again.